.travis.yml

```diff
@@ -3,16 +3,22 @@ rvm:
   - 2.4.1
 
 install:
- - cd site/
- - bundle install
+  - .ci/download_db.sh
+  - cd site/
+  - bundle install
+
+before_script:
+  - pluto update planet.ini
```

Comment:
Add a link to https://github.com/feedreader/jekyll-planet as a comment?

When googling for "planet jekyll" first few links are of [this website](https://planetjekyll.github.io/)